admin-portal/src/pages/aidPackage/assignSuppliers/assignSuppliers.tsx

```diff
@@ -179,9 +179,17 @@ export function AssignSuppliers({
           return (
             <>
               <tr {...row.getRowProps()} className="needs-row">
-                {row.cells.map((cell) => {
-                  return (
-                    <td {...cell.getCellProps()}>{cell.render("Cell")}</td>
+                {row.cells.map((cell, index) => {
+                  return index == 5 || index == 6 ? (
```

Comment:
Column order can be changed if we later decide to display more information. 

Using the `Cell` prop would be better approach:

https://stackoverflow.com/questions/57877482/react-table-render-component-inside-data

Reply:
Hey i changed it. Please review